specs/scrabble.spec.js

```diff
@@ -176,50 +176,59 @@ describe('Player', () => {
         // Assert
         expect(player.totalScore()).toBe(totalScore);
       });
-
     });
   });
 
   describe('hasWon', () => {
-    test.skip('returns false when score < 100', () => {
-
-
+    test('returns false when score < 100', () => {
+      const player = new Scrabble.Player('joe');
```

Comment:
It would work better to simply loop through verifying that `hashWon` returns false from a score of 0 to 99.  So you cover the edge cases.